app/scripts/directives/blockEditor.js

```diff
@@ -116,12 +116,17 @@ angular.module('confRegistrationWebApp')
           if (angular.isUndefined($scope.block.content)) {
             $scope.block.content = { 'choices': [] };
           } else if(angular.isUndefined($scope.block.content.choices)){
-            $scope.block.content.choices = [];
+              $scope.block.content.choices = [];
+          }
+          if (_.some($scope.block.content.choices, ['value', newOption])){
+            modalMessage.error('Error: Cannot use the same Multiple Choice Answer more than once.');
+          }
+          else {
+            $scope.block.content.choices.push({
```

Comment:
I removed it and placed it within the function but didn't notice any affect on behavior when tested.  Then I removed it completely and still didn't notice any affect.  Let me know if you want me to add it back into the function otherwise we can leave it out.

Reply:
Oh there's 2 of them. The other is here: https://github.com/CruGlobal/conf-registration-web/pull/561/files#diff-ed772b44f382f1184004ff83ef7994c2R80.

One gets called when you click the button (`ng-click`), the other gets called when you press enter (`ng-enter`).

Removing them both and moving them into the function should work. The input should be cleared only when the answer is successfully added.